<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>978</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>978</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File Format Converter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/main_icon.png</normaloff>icons/main_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>963</width>
        <height>461</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="toolTipDuration">
       <number>291</number>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>22</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/top_menu_50px.png</normaloff>icons/top_menu_50px.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <attribute name="toolTip">
        <string>Home</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>120</y>
          <width>51</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_up">
           <property name="maximumSize">
            <size>
             <width>57</width>
             <height>50</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Up</string>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton::hover{
	background-color: rgb(35,35,35);
};


background-color: rgb(3, 30, 35);
border-style: solid;
border-color: rgb(5,10,28);
border-width: 1px;
border-radius: 20px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/collapse_arrow_50px.png</normaloff>icons/collapse_arrow_50px.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_down">
           <property name="maximumSize">
            <size>
             <width>57</width>
             <height>50</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Down</string>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton::hover{
	background-color: rgb(35,35,35);
};


background-color: rgb(3, 30, 35);
border-style: solid;
border-color: rgb(5,10,28);
border-width: 1px;
border-radius: 20px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/expand_arrow_50px.png</normaloff>icons/expand_arrow_50px.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>661</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>7</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;List Of Files :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>30</number>
           </property>
           <property name="leftMargin">
            <number>45</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>87</width>
               <height>37</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>122</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Add New Folder</string>
             </property>
             <property name="styleSheet">
              <string notr="true">
QPushButton::hover{
	
	background-color: rgb(0, 68, 52);
	border-color: rgb(7, 106, 38);
	border-style: solid;
	border-width: 3px;
	border-radius: 6px;
	
};


background-color: rgb(7, 106, 38);

border-radius: 6px;



</string>
             </property>
             <property name="text">
              <string> Add Folder</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/Add Folder_50px.png</normaloff>icons/Add Folder_50px.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_add">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>37</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>122</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Add New File</string>
             </property>
             <property name="styleSheet">
              <string notr="true">

QPushButton::hover{
	
	background-color: rgb(0, 68, 52);
	border: 3px solid rgb(7, 106, 38);
	border-radius: 6px;
	
};


background-color: rgb(7, 106, 38);

border-radius: 6px;



</string>
             </property>
             <property name="text">
              <string> Add File</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/plus_50px.png</normaloff>icons/plus_50px.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QListWidget" name="listWidget">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Items</string>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 2px solid rgb(0,20,32);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>370</y>
          <width>260</width>
          <height>40</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>14</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_remove">
           <property name="minimumSize">
            <size>
             <width>103</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>122</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Remove Selected</string>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton::hover{
	
	background-color: rgb(210,0,0);
	border-color: rgb(3,3,3);
	border-style: solid;
	border-width: 1px;
	border-radius:5px;

};


background-color: rgb(181, 0, 0);
border-color: rgb(5,5,5);
border-style: solid;
border-width: 1px;
border-radius:4px;

</string>
           </property>
           <property name="text">
            <string> Remove</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/minus_50px.png</normaloff>icons/minus_50px.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear">
           <property name="minimumSize">
            <size>
             <width>103</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>122</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Clear All</string>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton::hover{
	
	background-color: rgb(200,0,0);
	border-color: rgb(3,3,3);
	border-style: solid;
	border-width: 1px;
	border-radius:5px;

};


background-color: rgb(181, 0, 0);
border-color: rgb(5,5,5);
border-style: solid;
border-width: 1px;
border-radius:4px;

</string>
           </property>
           <property name="text">
            <string> Clear All</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/clear_symbol_50px.png</normaloff>icons/clear_symbol_50px.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>90</y>
          <width>192</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_make_pdf">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton::hover{
	
	background-color: rgb(49, 49, 142);
	border-color: rgb(1, 1, 1);
	border-style: solid;
	border-width: 3px;
	border-radius:7px;
};



background-color: rgb(47, 47, 137);
border-color: rgb(1, 1, 1);
border-style: solid;
border-width: 1px;
border-radius:5px;
</string>
           </property>
           <property name="text">
            <string>IMG TO PDF</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/export_pdf_50px.png</normaloff>icons/export_pdf_50px.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_make_pdf_2">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton::hover{
	
	background-color: rgb(49, 49, 142);
	border-color: rgb(1, 1, 1);
	border-style: solid;
	border-width: 3px;
	border-radius:7px;
};



background-color: rgb(47, 47, 137);
border-color: rgb(1, 1, 1);
border-style: solid;
border-width: 1px;
border-radius:5px;
</string>
           </property>
           <property name="text">
            <string>DOCX TO PDF</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/export_pdf_50px.png</normaloff>icons/export_pdf_50px.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_make_pdf_3">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton::hover{
	
	background-color: rgb(49, 49, 142);
	border-color: rgb(1, 1, 1);
	border-style: solid;
	border-width: 3px;
	border-radius:7px;
};



background-color: rgb(47, 47, 137);
border-color: rgb(1, 1, 1);
border-style: solid;
border-width: 1px;
border-radius:5px;
</string>
           </property>
           <property name="text">
            <string>PDF TO DOCX</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/export_pdf_50px.png</normaloff>icons/export_pdf_50px.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/info_50px.png</normaloff>icons/info_50px.png</iconset>
       </attribute>
       <attribute name="title">
        <string>About</string>
       </attribute>
       <attribute name="toolTip">
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;strong&gt;Developers:&lt;/strong&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Rohan Godbole, Ashish Moghe, Chinmay Puranik, Soham Pathre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Icons Credit:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://icons8.com&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#0055ff;&quot;&gt;Icons8.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>283</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>81</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>188</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Technologies Used:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>191</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Python 3.10, PyQt5, PILLOW (PIL), PDF2DOCX, DOCX2PDF.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
